client/my-sites/plans/jetpack-plans/product-lightbox/constants.ts

```diff
@@ -25,24 +28,30 @@ export const PRODUCT_OPTIONS: Record< string, TranslateResult > = {
 	[ PLAN_JETPACK_SECURITY_T2_YEARLY ]: translate( '1TB (1000GB)', {
 		comment: 'GB here is the abbreviation for Gigabyte, and TB is abbreviation for Terabyte',
 	} ),
-	[ PRODUCT_JETPACK_SOCIAL_BASIC_BI_YEARLY ]: translate( 'Social Basic', {
-		comment: 'Get up to 1000 shares',
-	} ),
-	[ PRODUCT_JETPACK_SOCIAL_BASIC ]: translate( 'Social Basic', {
-		comment: 'Get up to 1000 shares',
-	} ),
-	[ PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY ]: translate( 'Social Basic', {
-		comment: 'Get up to 1000 shares',
-	} ),
-	[ PRODUCT_JETPACK_SOCIAL_ADVANCED_BI_YEARLY ]: translate( 'Social Advanced', {
-		comment: 'Get unlimited shares and share as a post by attaching images or videos.',
-	} ),
-	[ PRODUCT_JETPACK_SOCIAL_ADVANCED ]: translate( 'Social Advanced', {
-		comment: 'Get unlimited shares and share as a post by attaching images or videos.',
-	} ),
-	[ PRODUCT_JETPACK_SOCIAL_ADVANCED_MONTHLY ]: translate( 'Social Advanced', {
-		comment: 'Get unlimited shares and share as a post by attaching images or videos.',
```

Comment:
These translator comments seemed more like marketing comments so I removed them. Additionally we don't want to translate product names anyway